user_docs/en/userGuide.md

```diff
@@ -300,21 +300,21 @@ When add-ons are installed, the Add-on Store opens to the "Installed add-ons" ta
 #### Available add-ons {#AvailableAddons}
 When the window first opens, add-ons may take a few seconds to load.
 NVDA will read the name of the first add-on once the list of add-ons finishes loading.
-Available add-ons are listed alphabetically in a grid.
+Available add-ons are listed alphabetically in a multi-column list.
 To browse the list and find out about a specific add-on:
 
-1. Use the `arrow keys` or press the first letter of an add-on name to move around the list.
+1. Use the arrow keys or press the first letter of an add-on name to move around the list.
 1. Press `tab` once to move to a description of the currently selected add-on.
-1. Use the [reading keys](#ReadingText) or `arrow keys` to read the full description.
+1. Use the [reading keys](#ReadingText) or arrow keys to read the full description.
 1. Press `tab` to the "Actions" button, which can be used to install the add-on, among other actions.
 1. Press `tab` to "Other Details", which lists details such as the publisher, version and homepage.
-1. To return to the list of add-ons, press `alt+a`, or `shift+tab` back to the "Available add-ons" list.
+1. To return to the list of add-ons, press `alt+a`, or multiple times `shift+tab`.
 
 #### Searching for add-ons {#SearchingForAddons}
 As well as browsing all available add-ons, it is possible to filter the add-ons shown.
-To search, press `alt+s` to jump to the "search" field and type the text to search for.
-The search field can contain information including the add-on ID, display name, publisher, author or description.
-The list changes while typing the search terms.
+To search, press `alt+s` to jump to the "Search" field and type the text to search for.

```

Comment:
The label "Search" is capitalized in the GUI, so it should also be when you quote it here.

Reply:
Happy for the change of "search" to "Search", but did you intend to remove the next two lines?

Reply:
@Qchristensen - they were modified, the equivalent changes I've discussed here: https://github.com/nvaccess/nvda/pull/16618#discussion_r1616431530